.circleci/amplify_init.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh -xv
+echo $1
```

Comment:
What are we echoing here?

Reply:
The repo name, to make sure it picks up the correct repo. This script is used by both auth and api repos